docs/markdown/pipelines/exomeseq.md

```diff
@@ -1,38 +1,54 @@
+> **_Note_** The pipeline was recently rewritten to DSL2, which brought a significant amount of changes. This document details usage for nf-core/sarek >3.0. If you need to rerun older versions, please take a look [here](https://github.com/qbic-pipelines/pipeline-docs/blob/7f79a88458e05686216179ee5c791bb7a1f70f9b/docs/markdown/pipelines/exomeseq.md) and visit the official [nf-core documentation](https://nf-co.re/sarek/2.7.2). For older version, select the respective one in the drop-down menu on the right hand side.
+
+> :warning: In older versions parameters are not validated! It is easy to get command line flags wrong, please double check that the respective parameter exists in the version you are using!
+
+## Input csv file
 
-In order to use sarek, you will need to create your own input file. You can find more information on how to create your input file in the [sarek input documentation](https://nf-co.re/sarek/2.7.1/usage).
+### Starting from Fastq
 
-We usually start our analyses at QBiC from paired-end Fastq files. For these files, so you can take this `input.tsv` as example:
+In order to use sarek, you will need to create your own input file. You can find more information on how to create your input file in the [sarek input documentation](https://nf-co.re/sarek/3.1.2/usage#input-sample-sheet-configurations).
 
-```txt
-Patient1    XY  0   Sample1 Lane1   /path/to/Data/Sample1_L001_R1.fastq.gz  /path/to/Data/Sample1_L001_R2.fastq.gz
-Patient1    XY  0   Sample1 Lane2   /path/to/Data/Sample1_L002_R1.fastq.gz  /path/to/Data/Sample1_L002_R2.fastq.gz
-Patient1    XY  1   Sample2 Lane1   /path/to/Data/Sample2_L001_R1.fastq.gz  /path/to/Data/Sample2_L001_R2.fastq.gz
-Patient1    XY  1   Sample2 Lane2   /path/to/Data/Sample2_L002_R1.fastq.gz  /path/to/Data/Sample2_L002_R2.fastq.gz
-Patient1    XY  1   Sample2 Lane3   /path/to/Data/Sample2_L003_R1.fastq.gz  /path/to/Data/Sample2_L003_R2.fastq.gz
-Patient2    XY  0   Sample3 Lane1   /path/to/Data/Sample3_L001_R1.fastq.gz  /path/to/Data/Sample3_L001_R2.fastq.gz
-Patient2    XY  1   Sample4 Lane1   /path/to/Data/Sample4_L001_R1.fastq.gz  /path/to/Data/Sample4_L001_R2.fastq.gz
+We usually start our analyses at QBiC from paired-end fastq files. For these files, so you can take this `input.csv` as example:
+
+```csv
+patient,sex,status,sample,lane,fastq_1,fastq_2
+patient1,XX,0,normal_sample,lane_1,/path/to/Data/test_L001_1.fastq.gz,/path/to/Data/test_L001_2.fastq.gz
+patient1,XX,0,normal_sample,lane_2,/path/to/Data/test_L002_1.fastq.gz,/path/to/Data/test_L002_2.fastq.gz
+patient1,XX,0,normal_sample,lane_3,/path/to/Data/test_L003_1.fastq.gz,/path/to/Data/test_L003_2.fastq.gz
+patient1,XX,1,tumor_sample,lane_1,/path/to/Data/test2_L001_1.fastq.gz,/path/to/Data/test2_L001_2.fastq.gz
+patient1,XX,1,tumor_sample,lane_2,/path/to/Data/test2_L002_1.fastq.gz,/path/to/Data/test2_L002_2.fastq.gz
+patient1,XX,1,relapse_sample,lane_1,/path/to/Data/test3_L001_1.fastq.gz,/path/to/Data/test3_L001_2.fastq.gz
 ```
```

Comment:
```suggestion
```

Reply:
I would just leave the link to the docs on how to create the samplesheet so this does not need to be updated every time